bin/studio-common

```diff
@@ -17,6 +17,52 @@
 # limitations under the License.
 #
 
+# This value is automatically updated by Expeditor to reflect .hab-version
+export SUPPORTED_HAB_VERSION="0.34.1"
+
+# Ensure that the installed version of the hab toolchain is >= to the required version.
+#
+# Given the rapid development and release cadence of Habitat, we want to ensure that
+# developers are using a version of Habitat is is well tested against both this library
+# and the supporting automation (i.e. Expeditor builds).
+required_version="$SUPPORTED_HAB_VERSION"
+installed_version=$(hab -V | awk '{print $NF}' | cut -f1 -d/);
+
+MAJOR=$(echo "$installed_version"  | cut -f1 -d.);
+MINOR=$(echo "$installed_version"  | cut -f2 -d.);
+PATCH=$(echo "$installed_version"  | cut -f3 -d.);
+R_MAJOR=$(echo "$required_version" | cut -f1 -d.);
+R_MINOR=$(echo "$required_version" | cut -f2 -d.);
+R_PATCH=$(echo "$required_version" | cut -f3 -d.);
+
+need_upgrade=false;
+is_newer=false;
+
+[[ $PATCH -lt $R_PATCH ]] && need_upgrade=true;
```

Comment:
[nit] You added a `is_newer`and you are checking the minor and major but not the patch. This is a nit because if the version we have is not exactly the one that ci-studio-common has, we will have always a message, if newer or older. Not sure how I feel about that since the enforce version was supposed to be just for cases when we add functionality that an specific version of habitat has. Just something to consider, but I wont block this.

Reply:
ah, good catch.

Reply:
The logic here is that we DO want people using the specific version. We've seen in three different cases where breaking changes were made in patch or minor releases. So, it follows that we want to make sure people are using a well tested version.